index.bs

```diff
@@ -162,17 +156,21 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
   <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
 
 <h3 id="image-and-image-collection-brand">AVIF image and image collection brand</h3>
-Files that conform with the profile-independent restrictions in this document (sections [[#image-item-and-properties]] and [[#alpha-images]]) should include the brand
-<dfn value="" export="" for="AVIF Image brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
-<p>Files should also carry a compatible brand to identify the AVIF profile (see section [[#profiles]]), if any, with which the file complies.</p>
-<p>If <code>avif</code> is specified in the major_brand field of the FileTypeBox, the file extension should be ".avif".
+Files that conform with the profile-independent restrictions in this document (sections [[#image-item-and-properties]] and [[#alpha-images]]) should include in the [=compatible_brands=] field of the [=FileTypeBox=]:
+- the brand <dfn value="" export="" for="AVIF Image brand">avif</dfn> ,
+- the brand 'mif1' as recommended in [[!HEIF]],
```

Comment:
Doesn't "mif1" need to be enclosed in a "code" tag to avoid the Bikeshed dangling reference warning?

Reply:
I already added 'mif1' in the HEIF-defined property so Bikeshed does not complain. The code tag is used only to change the style of the text and does not change Bikeshed processing IIUC.

There is some history here: Bikeshed was written for CSS and I think in the CSS specs everything that was surrounded by single-quote was a property. That's why when we use single-quotes we have to link using the "property" type.